src/meta/src/hummock/compaction/picker/min_overlap_compaction_picker.rs

```diff
@@ -54,6 +54,7 @@ impl MinOverlappingPicker {
         level_handlers: &[LevelHandler],
     ) -> (Vec<SstableInfo>, Vec<SstableInfo>) {
         let mut scores = vec![];
+        let mut trivial_move_files = vec![];
```

Comment:
if we need this change after https://github.com/risingwavelabs/risingwave/pull/8996 ?